### PR TITLE
doc: stylish stash.md, #24717

### PR DESCRIPTION
**Pull request by patriknw**
Refs #24717

### CODE REVIEW

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala [hunk 1]
```diff
@@ -4,90 +4,107 @@
...
+  }
+
+  class DataAccess(
+      context: ActorContext[DataAccess.Command],
+      buffer: StashBuffer[DataAccess.Command],
+      id: String,
+      db: DB) {
+    import DataAccess._
+
+    private def init(): Behavior[Command] = {
```
Comment (helena) [inline akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala]: Strikes me more as `start()`, as start functions are typically Unit return types, where initialize functions take initialization params.

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala [hunk 2]
```diff
@@ -4,90 +4,107 @@
...
   }
 
   object DataAccess {
-    trait Command
+    sealed trait Command
     final case class Save(value: String, replyTo: ActorRef[Done]) extends Command
     final case class Get(replyTo: ActorRef[String]) extends Command
     private final case class InitialState(value: String) extends Command
     private final case object SaveSuccess extends Command
     private final case class DBError(cause: Throwable) extends Command
```
Comment (helena) [inline akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala]: Out of scope but not a command.

**Review by helena — APPROVED** [review]
For a style sample, LGTM - I added maybe one minor thought.

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala (hunk 2 above)
Comment (patriknw) [reply akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala]: The naming convention for incoming messages are `Command`. DBError i via pipeToSelf, but still an incoming message. See:
* https://doc.akka.io/docs/akka/snapshot/typed/style-guide.html#additional-naming-conventions
* https://doc.akka.io/docs/akka/snapshot/typed/style-guide.html#public-versus-private-messages

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala (hunk 1 above)
Comment (patriknw) [reply akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala]: ok, changed to `start`

**Pull request merged**